courses/beginner/10.md

```diff
@@ -48,207 +41,183 @@ control 'nginx-modules' do
   desc 'The required NGINX modules should be installed.'
 end
 
-control 'nginx-conf' do
+control 'nginx-conf-file' do
   impact 1.0
-  title 'NGINX configuration'
-  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
+  title 'NGINX configuration file'
+  desc 'The NGINX config file should exist.'
 end
 
-include_controls 'my_nginx_overlay'
-```
+control 'nginx-conf-perms' do
+  impact 1.0
+  title 'NGINX configuration permissions'
+  desc 'The NGINX config file should owned by root, be writable only by owner, and not writeable or and readable by others.'
+end
 
-```ruby
-# my_nginx_overlay
-control 'first-overlay-control'
-control 'second-overlay-control'
+control 'nginx-shell-access' do
+  impact 1.0
+  title 'NGINX shell access'
+  desc 'The NGINX shell access should be restricted to admin users.'
 ```
 
-In the example above, every time `my_nginx` profile is executed, all the controls from `my_nginx_overlay` are also executed. Therefore, the following controls would be executed:
+### Defining the Profile Dependency  
 
- - nginx-version
- - nginx-modules
- - nginx-conf
- - first-overlay-control
- - second-overlay-control
+For a profile to use controls from another profile, the dependency needs to be included in the `depends` section of the `inspec.yml` file. For example, you can develop my_nginx_overlay that uses controls from my_nginx profile. In that case, the `depends` section of `inspec.yml` of my_nginx_overlay should list the name and location of my_nginx. One way of declaring the dependency is:
 
-### Skipping a Control From a Profile
+::: code-tabs
+@tab my_nginx_overlay
+```yaml
+depends:
+  - name: my_nginx
+    path: ../my_nginx
+    # {path relative to the overlay}
```

Comment:
Can this go on the line above to the right? Just a nitpick

Reply:
I was thinking about this too tbh :)